brainfuck/parser.go

```diff
@@ -0,0 +1,42 @@
+package brainfuck
+
+type parser struct {
```

Comment:
minor. I think in this case this struct and parse func should have some doc. At least `parse` func to understand what's going on inside it.

Reply:
Fixed